clang/lib/Frontend/CompileJobCache.cpp

```diff
@@ -530,6 +543,15 @@ bool CompileJobCache::finishComputedResult(CompilerInstance &Clang,
     }
   }
 
+  bool UnsupportedOutput;
+  if (Error E = mayIngestNonVirtualOutputFromFileSystem(Clang).moveInto(
+          UnsupportedOutput)) {
+    reportCachingBackendError(Diags, std::move(E));
+    return false;
+  }
+  if (UnsupportedOutput)
+    SkipCache = true;
```

Comment:
Should remarks above this change to be emitted after `SkipCache` is updated here?

Reply:
Good catch! Done